privly.safariextension/Update.plist

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Extension Updates</key>
+	<array>
+		<dict>
+			<key>CFBundleIdentifier</key>
+			<string>com.privly.privly</string>
+			<key>Developer Identifier</key>
+			<string>34B3M269R6</string>
+			<key>CFBundleVersion</key>
+			<string>0.1.0</string>
+			<key>CFBundleShortVersionString</key>
+			<string>0.1.0</string>
+			<key>URL</key>
+			<string>http://privly.github.io/assets/extensions/safari/privly.safariextz</string>
```

Comment:
I have updated the code. Presently, the URL mentioned is not valid. We need to upload the extension to the URL specified.